.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,16 +4,16 @@ This pull request template helps you complete an application to the **Class of 2
 
 ## Checklist ✅
 
-- [ ] I have read the instructions on the README file before submitting my application. 
-- [ ] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
-- [ ] I have used the Markdown file template to add my information to the Year Book.
-- [ ] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
-- [ ] My application belongs to:
-  - [ ] **Tier 1** 📖: I added myself to the Year Book.
-  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
-  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
-- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
-- [ ] I understand I should not tag or add a reviewer to this Pull Request.
-- [ ] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
-- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
+- [x] I have read the instructions on the README file before submitting my application. 
+- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
+- [x] I have used the Markdown file template to add my information to the Year Book.
+- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
+- [x] My application belongs to:
+  - [x] **Tier 1** 📖: I added myself to the Year Book.
+  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
+  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
+- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
+- [x] I understand I should not tag or add a reviewer to this Pull Request.
+- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
+- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
```

Comment:
```suggestion
- [ ] I have read the instructions on the README file before submitting my application. 
- [ ] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [ ] I have used the Markdown file template to add my information to the Year Book.
- [ ] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [ ] My application belongs to:
  - [ ] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [ ] I understand I should not tag or add a reviewer to this Pull Request.
- [ ] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
```

Reply:
Do this and Your PR is ready to merged

Reply:
done